src/components/StoryList/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import * as MainSiteStyles from '../../globals/mainsiteGlobalStyles'
+
+interface HeaderProps {
+  title: string
+}
+
+export default function Header(props: HeaderProps) {
+  return (
+    <div
+      className={css`
+        background-color: #000;
```

Comment:
We may want to extract some of these colors into a global object. Thoughts @zeehang?

Reply:
Yeah - we can probably add colors to `GlobalStyles.tsx`